README.md

```diff
@@ -58,6 +58,9 @@ There is a live example API for testing purposes, [available here][sandbox].
 * Python (2.7, 3.4, 3.5, 3.6, 3.7)
 * Django (1.11, 2.0, 2.1)
 
+ We **highly recommend** and only officially support the latest release of each
```

Comment:
Wonder if this'd possibly confuse some folks. Perhaps "the latest *patch* release of each Python and Django series".

Reply:
Almost certainly. 😀

I just robbed it from the Django release notes, but yes... Incoming.